gemrb/GUIScripts/iwd/CharGen.py

```diff
@@ -384,7 +385,8 @@ def AcceptPress():
 	GemRB.SetPlayerName (MyChar, GemRB.GetToken ("CHARNAME"), 0)
 	GemRB.SetToken ("CHARNAME","")
 	GemRB.SetPlayerStat (MyChar, IE_XP, CommonTables.ClassSkills.GetValue (ClassName, "STARTXP"))
-
+	LUCommon.SetupSavingThrows (MyChar, Level=None)
```

Comment:
This actually runs? Either way, there's no need for the second parameter, since it has the same default.

Reply:
I'll adjust the code.